rm-datasource/src/main/java/com/alibaba/fescar/rm/datasource/undo/mysql/keyword/MySQLKeywordChecker.java

```diff
@@ -18,7 +18,7 @@
 
 import com.alibaba.fescar.rm.datasource.undo.KeywordChecker;
 
-import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang.StringUtils;
```

Comment:
why change the lang3 to lang?

Reply:
common-lang3 is not  fescar-all dependency transfer